marklogic-data-hub/src/main/resources/ml-modules/services/mlMappingFunctions.sjs

```diff
@@ -59,7 +59,7 @@ function getXpathFunctions() {
       let signature = xdmp.functionSignature(xpathFunctions[i]).replace("function", xdmp.functionName(xpathFunctions[i]));
       signature = signature.match(/fn:(.*?) as.*?/)[1];
       xpathFunction["signature"] = signature;
-      response[xdmp.functionName(xpathFunctions[i])] = xpathFunction;
+      response[String(xdmp.functionName(xpathFunctions[i])).replace("fn:","")] = xpathFunction;
```

Comment:
Can you update the getMappingFunctions.sjs test to do a verification on this - I'm thinking it would suffice to verify that "sum" exists  but not "fn:sum", with an assertion message stating why "sum" is there but not "fn:sum"

Reply:
Fixed in latest commit